lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/util/JsonUtils.java

```diff
@@ -158,12 +159,14 @@ public static JsonFactory jsonFactory()
     // due to the limits introduced by Jackson 2.15
     public static JsonFactoryBuilder jsonFactoryBuilder()
     {
+        // https://github.com/FasterXML/jackson-core/issues/1256
```

Comment:
it is weird that the link to https://github.com/FasterXML/jackson-core/issues/1256 is not next to the line with `recyclerPool`.
is this really what you wanted @electrum in https://github.com/trinodb/trino/pull/21361#discussion_r1548378187 ?